proto/attestation/attachment.proto

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2024 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.attestation.v1;
+
+import "proto/digest.proto";
+
+option go_package = "proto/oak/attestation/v1";
+option java_multiple_files = true;
+option java_package = "com.google.oak.attestation.v1";
+
+// The Transparent Release attachment for Oak Stage 0.
+message FirmwareAttachment {
+  // Maps number of vCPUs to measurement of the modified firmware binary.
+  // We cannot use map<int32, HexDigest> because it requires Rust std.
+  repeated int32 vcpu_counts = 1;
+  repeated HexDigest digests = 2;
+}
+
+// The Transparent Release attachment for Oak Containers Linux kernel.
+message KernelAttachment {
+  HexDigest binary = 1;
```

Comment:
Is this field still needed? I thought image and setup data is all we need

Reply:
Removed.